teams/teams-ps/teams/Get-CsOnlineTelephoneNumberOrder.md

```diff
@@ -13,82 +13,158 @@ ms.reviewer: julienp
 # Get-CsOnlineTelephoneNumberOrder
 
 ## SYNOPSIS
-Use the `Get-CsOnlineTelephoneNumberOrder` cmdlet to get the order report of a specific telephone number search order. The telephone numbers can then be used to set up calling features for users and services in your organization.
+Use the `Get-CsOnlineTelephoneNumberOrder` cmdlet to get the order report of a specific telephone number order. 
+
 
 ## SYNTAX
 
 ```
-Get-CsOnlineTelephoneNumberOrder [-OrderId] <OrderId> [<CommonParameters>]
+Get-CsOnlineTelephoneNumberOrder -OrderId <String> [-OrderType <String>]
+ [<CommonParameters>]
 ```
 
 ## DESCRIPTION
 
-Use the `Get-CsOnlineTelephoneNumberOrder` cmdlet to get the order report of a specific telephone number search order. The telephone numbers can then be used to set up calling features for users and services in your organization.
+This `Get-CsOnlineTelephoneNumberOrder` cmdlet can be used to get the status of specific telephone number orders. Currently supported orders for retrievals are: [New-CsOnlineTelephoneNumberOrder](https://learn.microsoft.com/powershell/module/teams/new-csonlinetelephonenumberorder), [New-CsOnlineDirectRoutingTelephoneNumberUploadOrder](https://learn.microsoft.com/powershell/module/teams/New-CsOnlineDirectRoutingTelephoneNumberUploadOrder) and [New-CsOnlineTelephoneNumberReleaseOrder] (https://learn.microsoft.com/powershell/module/teams/New-CsOnlineTelephoneNumberReleaseOrder). When the OrderType is not indicated, the cmdlet will default to a Search order. 
```

Comment:
Broken link: Remove extra space after `[New-CsOnlineTelephoneNumberReleaseOrder]`

@pavellatif perhaps we can make it more explicit too?
> Currently supported orders for retrievals are: Search ([New-CsOnlineTelephoneNumberOrder](https://learn.microsoft.com/powershell/module/teams/new-csonlinetelephonenumberorder)), DirectRoutingNumberCreation ([New-CsOnlineDirectRoutingTelephoneNumberUploadOrder](https://learn.microsoft.com/powershell/module/teams/New-CsOnlineDirectRoutingTelephoneNumberUploadOrder)) and Release ([New-CsOnlineTelephoneNumberReleaseOrder](https://learn.microsoft.com/powershell/module/teams/New-CsOnlineTelephoneNumberReleaseOrder)).

Reply:
updated